src/dbus_api/macros.rs

```diff
@@ -84,3 +84,14 @@ macro_rules! properties_footer {
         }
     };
 }
+
+macro_rules! pool_op_logging {
+    ($oper:tt, $arg:expr; $postoper:tt $(, $args:expr)*; $engine_op:expr) => {{
```

Comment:
As a final step, I'd recommend doing the same thing with `$arg` as you've done with `$args` (capture a repeating pattern) so we can accept multiple parameters to the format string in `$oper`. It may make the most sense to also rename `$arg` to `$pre_args`, rename `$args` to `$post_args`, and rename `$oper` to `$preoper` so that it's clear what all of these are doing.

Reply:
Done; I chose `$pre_oper` to keep with the underscore convention that seems to be going here

Reply:
Also, since the `info!()` call that prints the `$pre_args` message may have only one item to print, it would be ideal to only pass the comma if there are multiple `$pre_args` to print.